docs/recipes/LocalFirstDataWithPowerSync.md

```diff
@@ -0,0 +1,713 @@
+interface AppConfig {
+    // Existing config properties
+    supabaseUrl: string;
+    supabaseAnonKey: string;
+    powersyncUrl: string;
+}
+
+// Add the new properties to the config object
+export const APP_CONFIG: AppConfig = {
+    // Existing config values
+    supabaseUrl: '<<YOUR_SUPABASE_URL>>',
+    supabaseAnonKey: '<<YOUR_SUPABASE_ANON_KEY>>',
+    powersyncUrl: '<<YOUR_POWERSYNC_URL>>',
+};
+```
+
+:::danger
+These values are not secure, even inside a compiled app. You should **never** store sensitive information like API keys your app's config.
+
+Supabase implements [row-level security](https://supabase.com/docs/guides/auth/row-level-security), so sharing the anon key is not a security risk, but other backends may not have this feature, so be careful!
+:::
+
+:::info **Environment-Specific Configurations**
+If you have different configurations for production, development, and testing environments, you can use `config.dev.ts` or `config.prod.ts` files to store these configurations.
+:::
+
+## Installing SDK and Dependencies
+
+### Install necessary dependencies for PowerSync.
+
+Powersync [requires polyfills](https://github.com/powersync-ja/powersync-js/blob/main/packages/powersync-sdk-react-native/README.md#install-polyfills)
+to replace browser-specific APIs with their React Native equivalents. These are listed as peer-dependencies so we need
+to install them ourselves.
+
+```shell
+npx expo install \
+  @journeyapps/powersync-sdk-react-native \
+  @journeyapps/react-native-quick-sqlite \
+  react-native-fetch-api \
+  react-native-polyfill-globals \
+  react-native-url-polyfill \
+  text-encoding \
+  web-streams-polyfill@^3.2.1 \
+  base-64 \
+  react-native-get-random-values
+```
+
+:::info Update your native dependencies  
+The PowerSync SDK includes native modules so be sure to run `npx pod-install` after installing, and/or rebuild your
+android project.
+:::
+
+:::note
+At the time of writing the PowerSync SDK is not compatible with `web-streams-polyfill@4.0.0`, so we specify version `^3.2.1`. 
+:::
+
+### Optional - Install necessary dependencies for Supabase
+
+If using Supabase also install that now
+
+```shell
+npx expo intall @supabase/supabase-js
+```
+
+## Configuring Babel and Polyfills
+
+### Import polyfills at App entry
+
+Ensure polyfills are imported in your app's entry file, typically `App.tsx`:
+
+```ts
+import "react-native-polyfill-globals/auto";
+import "@azure/core-asynciterator-polyfill";
+```
+
+:::tip
+In a fresh Ignite app, this would be in `app/app.tsx`. and placed at the top of the list of imports, right after
+the Reactotron config.
+:::
+
+### Add Babel Plugin
+
+Update `babel.config.js` to include the `transform-async-generator-functions` plugin:
+
+```js
+/** @type {import('@babel/core').TransformOptions['plugins']} */
+const plugins = [
+        //... other plugins
+        '@babel/plugin-transform-async-generator-functions',  // <-- Add this
```

Comment:
You can put `// success-line` on the line before this one so that it highlights it green as an addition and calls it out a bit better